src/NHibernate.Test/NHibernate.Test.csproj

```diff
@@ -3248,6 +3250,9 @@
     <EmbeddedResource Include="NHSpecificTest\NH1291AnonExample\Mappings.hbm.xml" />
   </ItemGroup>
   <ItemGroup>
+    <EmbeddedResource Include="NHSpecificTest\NH3985\Mappings.hbm.xml">
+      <SubType>Designer</SubType>
```

Comment:
Yes, I didn't like that either, but Visual Studio seems to keep adding it. Not sure exactly what it means and why it gets added for some mapping files while not others.